src/scss/components/layout/_index.scss

```diff
@@ -68,9 +68,9 @@ a[href="#content"] {
 }
```

Comment:
Could you correct the spelling of the `translateX()` function in the `transform` value, please?

```css
transform: translateX(-50%);
```

Reply:
please check the changes in latest commit